src/assets/scss/components/theme-switcher.scss

```
@@ -22,13 +22,17 @@
 
 .theme-switcher__button {
 	box-shadow: var(--shadow-xs);
-	padding: 0.625rem 0.875rem;
+	padding: 0.625rem 0.675rem;
```

```suggestion
	padding: 0.625rem 0.875rem;
```
For mobile, tablet, and screens larger than 1200px, we can maintain the same padding as before.